index.html

```diff
@@ -1,14 +1,100 @@
+            <section id="awesome">I took these photos in this area!<br><br><br>
+            <img src="https://i.imgur.com/icyPfkv.png" alt="photo of a mountain goat standing on a rock formation in Vail, CO" class="goat";/>
+                    </section>
+            <section id="text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime, numquam commodi! Accusamus reiciendis distinctio quibusdam ipsa perferendis? Numquam perspiciatis pariatur quia ducimus impedit nobis, deleniti dignissimos ipsam assumenda omnis qui nemo saepe excepturi vero odit, similique voluptate. Sit, recusandae a assumenda unde sunt voluptates expedita reiciendis veniam eos temporibus perspiciatis doloremque quae! Similique aspernatur laborum nam quos. Iste quis modi quos est hic voluptatem libero id, aut in dolorem iure, officia enim harum unde quia repudiandae dolore quo....<br>
+                      <a href="continue-reading.html"><button id="reading">Continue Reading</button>  </section></a>
+        </div>
+<!--My photos wouldn't upload to VS Code, so I used imgur to upload all my own photos and use those urls to share my photos here-->
+<!--Each of these blog post links goes to the same post-->
+        <div id="sidebar">
+            <a href="blog-post.html"><button id="date">July 30</button></a>
+            <h4>Just Another Post</h4>
+            <section>lorem psum dolor sit amet consectetur adipisicing elit. Maxime, numquam commodi! Accusamus reiciendis distinctio quibusdam ipsa perferendis? Numquam perspiciatis<hr>
+            <a href="blog-post.html"><button id="date">July 28</button></a>
+            <h4>And Another Post</h4>
+            <section>lorem psum dolor sit amet consectetur adipisicing elit. Maxime, numquam commodi! Accusamus reiciendis distinctio quibusdam ipsa perferendis? Numquam perspiciatis<hr>
+            <a href="blog-post.html"><button id="date">July 24</button></a>
+            <h4>One More Post</h4>
+            <section>lorem psum dolor sit amet consectetur adipisicing elit. Maxime, numquam commodi! Accusamus<hr>
+            <h4>Something of Note</h4>
+            <img src="https://i.imgur.com/uxJLfFs.png" alt="Mount Crested Butte" class="side";/>
+             Quibusdam ipsa perferendis? Numquam perspiciatis pariatur quia ducimus impedit nobis, deleniti dignissimos ipsam assumenda, similique voluptate. Sit, recusandae a assumenda unde sunt voluptates 
+            <a href="learn-more.html"><br><button id="more">Learn More</button></a><hr>
+            <h4>Something of Less Note</h4>
+            <img src="https://i.imgur.com/SMwTgQ5.png" alt="Black Canyon of the Gunnison" class="side";/>
+            Quibusdam ipsa perferendis? Numquam perspiciatis pariatur quia ducimus impedit nobis, deleniti dignissimos ipsam assumenda omnis, similique voluptate. Sit, recusandae a assumenda unde sunt voluptates 
+            <a href="learn-more.html"><br><button id="more">Learn More</button></a>  
+        </div>

```

Comment:
We could add a little padding or margin on the right hand side of this so that it would give yourself a small gap between your right and left hand column on your page